### PR TITLE
Fixed the directory name and made some tweaks in the CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by renja-grotemeyer**
I changed `frontend` into `challenges` and made some changes to the CONTRIBUTING.md.

Btw. it looks like you are currently working on migrating to `next.js`, maybe you should also mention that.

### CODE REVIEW

**Comment by DarkIntaqt** [comment]
Thanks a lot for the PR! You just made this project a lot more contribution-friendly ^^
I will definitely mention the NextJS migration in the CONTRIBUTION.md.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,12 +1,40 @@
...
-Download/clone the repo and run 
+Thank you for considering contributing to this project! We welcome any contributions, big or small. Here are some guidelines to help you get started:
+
+## Reporting Issues
+
+If you encounter a bug or have a feature request, please create an issue in the GitHub repository. Be sure to provide a clear and detailed description of the problem, including any steps needed to reproduce it. 
+
+## Making Changes
+
+If you want to fix an issue or add a new feature, we encourage you to create a pull request (PR). Here are the steps to follow:
```
Comment (DarkIntaqt) [inline CONTRIBUTING.md:11]: I would probably add something like this. 
"If you are not sure whether your feature is desired, feel free to ask in an issue first. "

Comment (renja-grotemeyer) [reply CONTRIBUTING.md:11]: Sure, that sounds good

**Comment by renja-grotemeyer** [comment]
OK, I just rewrote it.

**Review by DarkIntaqt — APPROVED** [review]
LGTM, thanks a lot

**Pull request merged**